create-doks-with-terraform-flux/README.md

```diff
@@ -1 +1,470 @@
+# Gitops stack using DOKS and Flux CD
 
+This blueprint will guide you step by step on how to spin up a single DOKS (DigitalOcean Kubernetes) cluster and Flux CD for managing application deployments in a GitOps fashion. In the end we will also tell [Flux CD](https://fluxcd.io) to perform a basic deployment of the now ubiquitous `busybox` Docker application.
+
+[Terraform](https://www.terraform.io) was chosen to write `infrastructure as code` using declarative configuration files which allows for concise descriptions of resources using blocks, arguments, and expressions. In our guide it will be responsible with spinning up the DOKS (DigitalOcean Kubernetes) cluster as well as [Flux CD](https://fluxcd.io).
```

Comment:
```suggestion
We use [Terraform](https://www.terraform.io) to write `infrastructure as code` using declarative configuration files which allows for concise descriptions of resources using blocks, arguments, and expressions. It will be responsible with spinning up the DOKS (DigitalOcean Kubernetes) cluster as well as Flux.
```

Reply:
Thanks for the suggestion. Fixed in the latest commit.